### ISSUE TITLE
UI for Google Sign in

### ISSUE THREAD

**Issue by alena1108** [rancher/rancher#20269]
This ticket is to keep track of the UI component for https://github.com/rancher/rancher/issues/1411

**Comment by davidnuzik** [comment]
Version: master-head (v2.3) (7/8/19) (latest UI)

**Step 2 first copy to clipboard includes http:// protocol** - is there a way to prevent this? We should only be copying the IP/rancher domain to clipboard not http.
<img width="555" alt="Screen Shot 2019-07-08 at 1 16 04 PM" src="https://user-images.githubusercontent.com/45179589/60839683-9c343b80-a182-11e9-8020-b9f6585faf16.png">

When I paste the content I get this in the Google UI (and have to manually remove the protocol)
<img width="465" alt="Screen Shot 2019-07-08 at 1 20 26 PM" src="https://user-images.githubusercontent.com/45179589/60839933-35635200-a183-11e9-9a97-e12550c57d12.png">

Also Google requires proper domains/fqdns you cannot use an IP address. It would be nice to warn the user about this in my opinion.

More testing to be done later, just noticed this.

**Comment by davidnuzik** [comment]
https://github.com/rancher/rancher/issues/21511 fixes some en-us translations (help text, instructions, etc).

I have tested basics, next up is to test with various users, some RBAC checks, etc.

**Comment by davidnuzik** [comment]
Version: master-head (7/16/19)

Initial testing has been completed. Further testing will be more thorough following our internal test plan.

I was successfully able to follow the steps outlined and set up Google auth and leverage it. I made some tweaks to instructions in issue #21511 and this is now merged in. This should help to make things a little more clear to end users. Once we have docs in, we may tweak this again later especially with regards to step four which has some extra steps in various areas.
